apps/back-office/src/app/(dashboard)/calendar/components/availability/AppointmentSidebar.tsx

```
@@ -77,8 +81,140 @@
+  // State for user's clinician ID
+  const [userClinicianId, setUserClinicianId] = useState<string | null>(null);
+
+  // Fetch clinician ID for non-admin users
+  useEffect(() => {
+    const fetchClinicianId = async () => {
+      if (isClinician && !isAdmin && userId) {
+        try {
+          const response = await fetch(`/api/clinician?userId=${userId}`);
+          if (response.ok) {
+            const data = await response.json();
+            if (data && data.id) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)